yarn.lock

```diff
@@ -7607,6 +7607,21 @@ beachball@^1.11.7:
     semver "^6.1.1"
     yargs-parser "^13.1.0"
 
+beachball@^1.29.0:
```

Comment:
Why didn't this replace the one above in the lock file?

Reply:
Also going to remove unnecessary beachball sections from package.json files